robottelo/cli/repository.py

```diff
@@ -13,8 +13,10 @@
 
     create                        Create a repository
     delete                        Destroy a repository
+    export                        Export a repository
     info                          Show a repository
     list                          List of repositories
+    remove-content
```

Comment:
@pondrejk if this is what hammer gave, so you can to leave as it is, and please report it. you can include issue in http://projects.theforeman.org/projects/hammer-cli/issues (if isn't there yet)